spec/services/browser_support_spec.rb

```diff
@@ -0,0 +1,120 @@
+require 'rails_helper'
+
+RSpec.describe BrowserSupport do
+  before do
+    BrowserSupport.cache.clear
+    BrowserSupport.instance_variable_set(:@browser_support_config, nil)
+    BrowserSupport.instance_variable_set(:@matchers, nil)
```

Comment:
maybe we add a `BrowserSupport.clear!` method and comment that it's intended for specs

Reply:
> maybe we add a `BrowserSupport.clear!` method and comment that it's intended for specs

Yeah, I had considered it (`BrowserCache` does similar). I wasn't a huge fan of tailoring the implementation to suit the specs, but I think it's probably fine to expose as part of the public interface.

Reply:
Added method in d20c79c79. In the process, I discovered the memoization for `browser_support_config` wasn't working as expected 🙈